doc/dev_manual/image_wrappers.rst

```diff
@@ -45,6 +45,18 @@ Once you have an object, you can load data directly contained in a
 
      >>> img.update_keywords(kw_dict)
 
+``AstroImage`` objects also support ``dask`` arrays::
+
+  >>> import dask.array as da
+  >>> data_dk = da.from_array(data, chunks=(100, 100))
```

Comment:
I just learned from @Cadair that providing a `name=` is very important, so that `dask` doesn't try to generate a name from data hash and eats up all the computing time.